travvy/src/components/Profile.vue

```diff
@@ -69,7 +71,44 @@
         this.$store.dispatch('setActivity', user.Preferred_Activity_Type)
         this.$store.dispatch('setName', user.Name)
 
-      }
+      },
+      async updateUserProfile(){
+        console.log(this.email)
+        console.log(this.age)
+        console.log(this.gender)
+        console.log(this.instagram_username)
+        console.log(this.preferred_activity_type)
+        console.log(this.errors)
+
+
+      try {
+        const response = await AuthenticationService.onboarding({
+          "email": this.email,
+          "age": this.age,
+          "gender": this.gender,
+          "instagram_username": this.instagram_username,
+          "preferred_activity_type": this.preferred_activity_type,
```

Comment:
Next sprint we can insert the activity type into the User_type table to be able to extract it properly when searching